CATALOG.md

```diff
@@ -35,3 +35,20 @@ nativefier https://www.udemy.com/
 ### Notes
 
 Most videos will work, but to be sure everything works you are better off using the `--widevine` version AND signing the app afterwards. See this post: [#1147 (comment)](https://github.com/nativefier/nativefier/issues/1147#issuecomment-828750362)
+
+
+## Spotify
+
+```
+nativefier https://play.spotify.com/
+--internal-urls ".*" 
```

Comment:
I think this is a little too expansive for internal-urls. Was there a reason that it needed to catch any url?

Reply:
certain login methods and the login page itself need to be included in the internal urls or you won't be able to log in with google, apple or facebook. Those could be added one by one

Reply:
I see that there is already logic to handle common login pages as internal, so the only thing that needs to be here is `accounts.spotify.com` the base url should also be changed to open.spotify.com since it's just a redirect

Reply:
Apple, Google, and Facebook should all get caught be our internal login pages filter. After testing this without any `--internal-urls` passed, everything still seems to work fine for me. So I'll be removing it in my update to address the below comment.